shared/src/commonMain/kotlin/co/touchlab/kampstarter/models/BreedModel.kt

```diff
@@ -39,39 +42,38 @@ class BreedModel(private val viewUpdate: (ItemDataSummary) -> Unit,
         }
     }
 
-    fun getBreedsFromNetwork() {
+    fun getBreedsFromNetwork(): Job {
         fun isBreedListStale(currentTimeMS: Long): Boolean {
             val lastDownloadTimeMS = settings.getLong(DB_TIMESTAMP_KEY, 0)
             val oneHourMS = 60 * 60 * 1000
             return (lastDownloadTimeMS + oneHourMS < currentTimeMS)
         }
 
         val currentTimeMS = currentTimeMillis()
+        var job: Job = Job().apply { complete() }
```

Comment:
You can avoid making this a var if you allocate the default one in an else branch.